resources/nn/tiny-yolo/tiny-yolo.json

```diff
@@ -0,0 +1,43 @@
+{
+    "tensors":
+    [
+        {       
+            "output_tensor_name": "out",
+            "output_dimensions": [1, 24, 26, 26],
+            "output_entry_iteration_index": 0,
+            "output_properties_dimensions": [0],
+            "property_key_mapping":
+            [
+
+            ],
+            "property_value_mapping":
+            [
+        
+            ],
+            "output_properties_type": "f16"
+        },
+	    {       
+            "output_tensor_name": "out2",
+            "output_dimensions": [1, 24, 13, 13],
+            "output_entry_iteration_index": 0,
+            "output_properties_dimensions": [0],
+            "property_key_mapping":
+            [
+            ],
+            "property_value_mapping":
+            [
+            ],
+            "output_properties_type": "f16"
+       }
+    ],
+    "mappings":
+    {
+        "labels":
+        [
+            "mask",
+            "no mask",
+            "no mask2"
```

Comment:
Should we rename the model name `tiny-yolo` to include something about the face mask?

Reply:
good question. I think we should remove it, since it's a custom network, and maybe add the official one ?

Reply:
thoughts @Luxonis-Brandon  ?

Reply:
Yes agreed.